kml/edit-kml-ground-overlay/README.md

```diff
@@ -0,0 +1,39 @@
+# Edit KML ground overlay
+
+Edit the values of a KML ground overlay.
+
+![Edit KML ground overlay](EditKMLGroundOverlay.png)
+
+## Use case
+
+KML ground overlays are used for showing aerial imagery, symbology, or other images draped over a scene. Changing the geometry, rotation, and other attributes of a ground overlay after it has been loaded allows for live editing.  For example, editing the geometry and opacity of a historical image draped over present day satellite imagery makes it possible to view change over time.
+
+## How to use the sample
+
+Use the slider to adjust the opacity of the ground overlay.
+
+## How it works
+
+1. Create an `Envelope` defining the geometry of the overlay.
+2. Create a `KmlIcon` using a `Uri` linking to an image.
+3. Create a `KmlGroundOverlay` using the envelope and icon.
+4. Use `kmlGroundOverlay.setRotation()` to set the rotation of the ground overlay.
```

Comment:
Is this super common that it needs to be listed here? I would think the extent would usually get the correct placement.

Reply:
Good point, it's just a small fix for the data used here, so it can go.